UnityProject/Assets/ScriptableObjects/Learning/Tips.asset

```diff
@@ -0,0 +1,131 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: ba683b5032c8b9145a8f076c69600e2e, type: 3}
+  m_Name: Tips
+  m_EditorClassIdentifier: 
+  Tips:
+  - <color=red>Security</color> personnel heal a small amount of brute damage by
+    eating donuts.
+  - Play as a <color=purple>Jaintor</color> if you're new. You have a clear objective
+    and role but there isn't expected much from you.
+  - Hold shift then click on an object to quickly examine it.
+  - Always be prepared before entering gateways or stepping into teleporters. You
+    never know what's on the other side.
+  - Flashlights emit light even when in your pocket. You don't have to hold them
+    to light your way unlike other games.
+  - Face huggers cannot lock themselves on your face if you have a mask on.
```

Comment:
```suggestion
  - Facehuggers cannot lock themselves onto your face if you have a mask on.
```

Reply:
it only works for some mask though :eyes: